Library/Homebrew/formula.rb

```diff
@@ -1614,6 +1625,16 @@ class << self
     # @private
     attr_reader :plist_manual
 
+    # The block evaluated by `pour_bottle?` to decide whether to use a bottle
+    # or build from source.
+    # @private
+    attr_reader :pour_bottle_block
+
+    # If `pour_bottle?` returns `false` the user-visible reason to display for
+    # why they cannot use the bottle.
+    # @private
+    attr_reader :pour_bottle_reason
```

Comment:
Some variation of `pour_bottle_impossible` or `bottle_unavailable_reason` if you're not wed to the `pour_*` wording seems okay to me.